routers/api/v1/api.go

```diff
@@ -22,6 +22,7 @@
 //
 //     Security:
 //     - BasicAuth :
+//     - ReverseProxyAuth :
```

Comment:
I don't think we can or should be putting the ReverseProxyAuth in the swagger. It's not something that a user necessarily sets and it's not going to be an available authentication method for most servers.

```suggestion
```

Reply:
ReverseProxyAuth removed from swagger as suggested.